### PR TITLE
Added hotkeys option to make_action

### PR DESCRIPTION
**Pull request by cblogichub**

### CODE REVIEW

File: plugin/LHUB.py [hunk 1]
```diff
@@ -66,7 +66,7 @@
 user_config_file = "logichub_tools.ini"
 
 # Will be updated if enabled via the config file
-debug_enabled = False
+debug_enabled = True
```
Comment (deathbywedgie) [inline plugin/LHUB.py:69]: This just sets the default. This doesn't need to be changed, even when testing. If you want to turn on debug logging (which currently isn't used by a lot), set "debug_output_enabled" to true in logichub_tools.ini

File: plugin/LHUB.py [hunk 2]
```diff
@@ -833,13 +833,18 @@ def set_status_bar_display(self):
         # Set status bar text and/or logo
         self.print_in_menu(self.status)
 
-    def make_action(self, name, action, action_id=None, menu_depth=1, alternate=False, terminal=False, text_color=None):
+    def make_action(self, name, action, action_id=None, menu_depth=1, alternate=False, terminal=False, text_color=None, keycmd=False):
         menu_name = name
+        shortcut_key = ''
+        if keycmd:
+        	shortcut_key = keycmd
```
Comment (deathbywedgie) [inline plugin/LHUB.py:840]: This doesn't appear to do anything. Does that mean a wrong variable was used somewhere else, or was this early code that just needs to come back out?

File: plugin/LHUB.py [hunk 3]
```diff
@@ -854,7 +859,7 @@ def make_action(self, name, action, action_id=None, menu_depth=1, alternate=Fals
         _var = action_tuple(action_id, name, action)
         self.action_list[action_id] = _var
         terminal = str(terminal).lower()
-        self.print_in_menu(f'{menu_name} | {action_string} bash="{self.script_name}" param1="{action_id}" terminal={terminal}')
+        self.print_in_menu(f'{menu_name} | {action_string} | bash="{self.script_name}" | param1="{action_id}" | terminal={terminal} ')
```
Comment (deathbywedgie) [inline plugin/LHUB.py:862]: The old line always worked fine in BitBar, but I haven't finished reviewing everything in BitBar to see if anything like this has changed. Was this necessary in order by make the shortcut keys work?

File: plugin/LHUB.py [hunk 4]
```diff
@@ -833,13 +833,18 @@ def set_status_bar_display(self):
...
+        shortcut_key = ''
+        if keycmd:
+        	shortcut_key = keycmd
         if menu_depth:
             menu_name = '--' * menu_depth + ' ' + menu_name
         action_string = ''
         if alternate:
             action_string = action_string + ' alternate=true'
+        if keycmd:
+        	action_string += ' key=' + keycmd
```
Comment (deathbywedgie) [inline plugin/LHUB.py:847]: Tab indentation? You're better than this! 😆

File: plugin/LHUB.py (hunk 3 above)
Comment (cblogichub) [reply plugin/LHUB.py:862]: It wasn't, but I did it to be in alignment with their rules.

File: plugin/LHUB.py (hunk 2 above)
Comment (cblogichub) [reply plugin/LHUB.py]: Removed

File: plugin/LHUB.py (hunk 1 above)
Comment (cblogichub) [reply plugin/LHUB.py]: Corrected

**Pull request merged**